src/main/java/frc/team997/lib/math/PeakDetectionFilter.java

```diff
@@ -165,22 +159,15 @@ public int calculate(double value) {
         return returnValue;
     }
 
-    /**
-     * Resets the history of the filter. This will not return any more peaks for at least {@code
-     * window - 1} more calls.
-     */
+    @Override
```

Comment:
This should still have docs, but it can use the @inheritDoc annotation to inherit from Filter instead of rewriting the doc